CHANGES.rst

```diff
@@ -32,13 +34,6 @@ Changelog
 - Fix Zope DateTime convert to also handle the datetime.date type [ewohnlich]
 
 
-3.0.1 (2019-01-28)
-------------------
-
-- Fix sortable_title search issue
-  [ewohnlich]
```

Comment:
Please don't remove previous entries.

Reply:
The only reason I removed it is because the exact entry was in there twice, consecutively. And both by me. I can add it back though.

Reply:
Okay. You don't need to add it again. But this should be done in a separate commit.